### PR TITLE
fix: link to view profile

### PR DESCRIPTION
**Pull request by blushi**
## Description

ref: https://github.com/regen-network/regen-web/issues/2248#issuecomment-1931144892

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. Go to https://deploy-preview-2280--regen-marketplace.netlify.app/ and connect with a **web2** account (no wallet address)
2. Go to edit profile and click on "view profile"
3. It should go back to your profile page as expected

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-marketplace/src/pages/ProfileEdit/ProfileEdit.ViewProfile.tsx [hunk 1]
```diff
@@ -21,9 +21,9 @@ export const ViewProfileButton = ({ setIsWarningModalOpen }: Props) => {
     <OutlinedButton
       onClick={() => {
         if (isDirtyRef?.current) {
-          setIsWarningModalOpen('/profile/portfolio');
+          setIsWarningModalOpen('/profile');
```
Comment (blushi) [inline web-marketplace/src/pages/ProfileEdit/ProfileEdit.ViewProfile.tsx:24]: /profile/portfolio is only available to web3 users which is why it was displaying a warning to connect a wallet address
updating to the generic /profile path

**Comment by blushi** [comment]
@erikalogie @clevinson see testing instructions above

**Comment by erikalogie** [comment]
This is what is happening for me: https://monosnap.com/file/mVEXAqd1SUyNAX944OIfoVqW0zpRPc

**Comment by blushi** [comment]
@erikalogie yes this is precisely fixing this issue, looking at your video, you're testing on dev.app.regen.network where the bug is still there, please use the deploy preview link instead: https://deploy-preview-2280--regen-marketplace.netlify.app/

**Comment by clevinson** [comment]
This deploy preview doesn't work for me (on Safari) when clicking on email log in:
- I type a valid email
- press "Log In"
- Button blinks, but nothing else happens
- When I open console, I get a the following error: `403 forbidden: https://api-staging.regen.network/marketplace/v1/auth/passcode`

Logging in with Google works as expected and fixes the bug.

My guess is that the issue has something to do with api-staging.regen.network not allowing requests from domains other than dev.app.regen.network? So may be this is a non-issue. But I wanted to flag anyway incase it is indeed a problem that needs resolving.

**Comment by aaronc** [comment]
For me I always get redirected to dev.app.regen.network when I try to test this. I don't seem to be able to login with that deploy preview to actually test.

**Review by aaronc — APPROVED** [review]
Works in Brave

**Pull request merged**